### PR TITLE
store: kill setUbuntuStoreHeaders

### PR DESCRIPTION
**Pull request by chipaca**

### CODE REVIEW

File: store/store.go [hunk 1]
```diff
@@ -127,6 +127,16 @@ type Store struct {
 	suggestedCurrency string
 }
 
+func reqFailed(resp *http.Response, msg string) error {
```
Comment (pedronis) [inline store/store.go]: respError ? respToError?

File: store/store.go [hunk 2]
```diff
@@ -127,6 +127,16 @@ type Store struct {
...
+	tpl := "cannot %s: got unexpected HTTP status code %d via %s to %s"
```
Comment (pedronis) [inline store/store.go]: would use %q for the last bit

**Comment by pedronis** [comment]
small suggestions otherwise lgtm

**Comment by pete-woods** [comment]
This looks good to me, have agreed that bubbling up the BuyOptions changes should come after

**Pull request merged**